modules/glimpse/chunk/main.nf

```diff
@@ -0,0 +1,43 @@
+process GLIMPSE_CHUNK {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::glimpse-bio=1.1.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:1.1.1--h2ce4488_2':
+        'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
+
+    input:
+    tuple val(meta), path(reference_panel)
+    val region
+
+    output:
+    tuple val(meta), path("*.txt"), emit: chunk_chr
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix          = task.ext.prefix ?: "${meta.id}"
+    def thread_command  =                    "--thread $task.cpus"
```

Comment:
I would suggest just putting `--thread $task.cpus` in the command itself, since you don't really check for anything here and it will be the same everytime you run it.